scripts/check_bot_review.py

```
@@ -765,13 +789,14 @@ def main(argv: list[str] | None = None) -> int:
         verified_code, _ = check_run_verdict(
             owner, repo, head_sha, token,
         )
-        if verified_code == EXIT_STUB and exit_code == EXIT_OK:
+        if verified_code == EXIT_STUB and exit_code == EXIT_OK and not RECONCILE_403_OCCURRED:
```

Ruff (0.16.3)

[error] Local variable `RECONCILE_403_OCCURRED` referenced before assignment

(F823)